docs/custom-images-contract.rst

```diff
@@ -24,15 +24,33 @@ Contract statements
 
     docker run myscrapyimage start-crawl
 
-2. Docker image should be able to return a spiders list via ``list-spiders`` command without arguments.
-   ``list-spiders`` should be :ref:`executable and located on the search path <scripts-example>`.
+2. Docker image should be able to return its metadata via ``shub-image-info`` command without arguments.
+   ``shub-image-info`` should be :ref:`executable and located on the search path <scripts-example>`.
```

Comment:
Prepared changes for CasperJS example https://github.com/scrapinghub/custom-images-examples/pull/9/files